packages/ckeditor5-minimap/src/utils.ts

```diff
@@ -88,29 +90,26 @@ export function getPageStyles() {
  * TODO
```

Comment:
Either add comments or remove TODO. IMO, it shouldn't be in API docs.
The same for functions below